cmd/gotk/get_source_bucket.go

```diff
@@ -49,36 +52,39 @@ func getSourceBucketCmdRun(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader([]string{"Name", "Revision", "Ready Status", "Message", "Namespace"})
+	table.SetRowLine(true)
```

Comment:
With the below change, this can be moved above the loop that goes through the `list.Items`.